### PR TITLE
feat: add selectMatchingValues action

### PR DESCRIPTION
**Pull request by haxxmaxx**
Adding this action would solve a bug and make sure we have full feature parity.

I also want to revert the fix I made for dates, since it only covers some cases. Not sure if that is possible since it was delievered in MS36. Will look into how this should be released.

### CODE REVIEW

File: src/utils/actions.js [hunk 1]
```diff
@@ -100,6 +100,18 @@ const actions = [
     },
     requiredInput: ['field', 'value', 'softLock'],
   },
+  {
+    value: 'selectMatchingValues',
+    translation: '$Select matching values in a field',
```
Comment (niekvanstaveren) [inline src/utils/actions.js:105]: Needs proper translation string

Comment (haxxmaxx) [reply src/utils/actions.js:105]: Yup. started the discussion in the PS channel to come up with a good name.

**Comment by haxxmaxx** [comment]
@Caele I can't see any problems with backwards compatibility, since we are not changing anything to existing actions. Everything else looks good from an architectural pov?

**Comment by Caele** [comment]
> @Caele I can't see any problems with backwards compatibility, since we are not changing anything to existing actions. Everything else looks good from an architectural pov?

What happens if you use the new action and then downgrade?

**Comment by haxxmaxx** [comment]
> > @Caele I can't see any problems with backwards compatibility, since we are not changing anything to existing actions. Everything else looks good from an architectural pov?
> 
> What happens if you use the new action and then downgrade?

Good question, I'll try it

**Comment by haxxmaxx** [comment]
@Caele the action shows up, but missing a lot of info, and does, and does nothing when clicking. What do we want to happen? Not showing it at all might lead to some severe misunderstandings when the button just does nothing without any feedback in older versions.

**Pull request merged**